next.config.js

```diff
@@ -35,20 +35,16 @@ const publicRuntimeConfig = {
   MAX_UPLOADING_FILE_SIZE: process.env.MAX_UPLOADING_FILE_SIZE || '256KB',
   TOTAL_MAX_UPLOADING_FILES_SIZES: process.env.TOTAL_MAX_UPLOADING_FILES_SIZES || '5M',
   LENGTH_KEY: process.env.LENGTH_KEY || 2048,
+  WORKER_DECRYPTION_BUFFER_SIZE: process.env.WORKER_DECRYPTION_BUFFER_SIZE || 10,
```

Comment:
Why we have constant variable WORKER_DECRYPTION_BUFFER_SIZE and environment variable WORKER_DECRYPTION_BUFFER_SIZE at one time?

Reply:
My mistake. I'll fix it